scripts.js

```diff
@@ -1,84 +1,110 @@
+const storeApp = {};
 
-// this is the store data. normally we would be grabbing data from elsewhere (an external api), but this will represent the data model! 
-  const totalInventory = [
-    {
-      title: 'Bowie Tee',
-      url: 'images/bowie.jpg',
-      price: 19.99,
-      stock: 4,
-    }, 
-    {
-      title: 'Don\'t Know Tee',
-      url: 'images/dontevenknow.jpg',
-      price: 22.50,
-      stock: 8,
-    }, 
-    {
-      title: 'Doughnut Jean Jacket',
-      url: 'images/doughnut.jpg',
-      price: 59.00,
-      stock: 5,
-    }, 
-    {
-      title: 'Journey Tee',
-      url: 'images/journey.jpg',
-      price: 22.99,
-      stock: 6,
-    }, 
-    {
-      title: 'Skeleton Jean Jacket',
-      url: 'images/someurl.jpg',
-      price: 30.00,
-      stock: 0,
-    }, 
-    {
-      title: 'Skeleton Hand Tee',
-      url: 'images/skeleton.jpg',
-      price: 30.00,
-      stock: 10,
-    }, 
-    {
-      title: 'HackerYou Hoodie',
-      price: 50.00,
-      stock: 4,
-    }, 
-  ]
+storeApp.totalInventory = [
+  {
+    title: 'Bowie Tee',
+    url: 'images/bowie.jpg',
+    price: 19.99,
+    stock: 4,
+  },
+  {
+    title: 'Don\'t Know Tee',
+    url: 'images/dontevenknow.jpg',
+    price: 22.50,
+    stock: 8,
+  },
+  {
+    title: 'Doughnut Jean Jacket',
+    url: 'images/doughnut.jpg',
```

Comment:
Great job with everything! Why don't we try using this image instead?
```suggestion
    url: 'images/Sprinkledoughnut.jpg',
```